Fix CLI Documentation (#511)
opensearch-project/opensearch-py-ml: docs/source/cli/create_connector.rst

.. _cli.create_connector:

==================
Create a connector
==================

Description
~~~~~~~~~~~

Use the `connector create` command to add a connector to either a self-managed OpenSearch cluster or Amazon OpenSearch Service. You can create a connector interactively by following prompts or by using a preconfigured YAML file.

> To create a connector in self-managed OpenSearch, you need to:

Reviewer: ```suggestion
To create a connector in self-managed OpenSearch, do the following:
```

* Select from supported connectors and their associated models
* Provide necessary configuration parameters (for example, API key)

> OpenSearch Service automatically handles AWS-specific setup including:

Reviewer: ```suggestion
OpenSearch Service automatically handles AWS-specific setup, including the following components:
```

  * IAM role creation and configuration
  * Backend role mapping
  * Secret creation

> To create a connector in OpenSearch Service, you need to:

Reviewer: ```suggestion
To create a connector in OpenSearch Service, do the following:
```
